spec/integration/explicit_encryption_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+
+describe 'Explicit Encryption' do
+  require_libmongocrypt
+
+  let(:client) { ClientRegistry.instance.new_local_client(['localhost:27017']) }
+  let(:key_vault_namespace) { 'test.keys' }
+
+  let(:client_encryption_opts) do
+    {
+      kms_providers: {
+        local: { key: Base64.encode64("ru\xfe\x00" * 24) }
```

Comment:
The spec in https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/client-side-encryption.rst#kmsproviders specifies that key is a 96 byte string. Why is it base64 encoded here?

Reply:
This is a good question! It's actually in the FLE tutorial on the documentation page (https://docs.mongodb.com/manual/tutorial/manage-client-side-encryption-data-keys/)

> Configuring client-side field level encryption for a locally managed key requires specifying a base64-encoded 96-byte string with no line breaks.

I'm going to make a SPEC ticket for this right now.

Reply:
spec ticket: https://jira.mongodb.org/browse/SPEC-1501